modules/core/src/main/java/org/apache/ignite/configuration/TransactionConfiguration.java

```diff
@@ -457,9 +454,6 @@ public TransactionConfiguration setTxAwareQueriesEnabled(boolean txAwareQueriesE
     private static String[] transientSerializableFields(IgniteProductVersion ver) {
         ArrayList<String> transients = new ArrayList<>(2);
 
-        if (TX_PME_TIMEOUT_SINCE.compareToIgnoreTimestamp(ver) >= 0)
```

Comment:
Looks like `transientSerializableFields` not used at all.
We can remove this method and two constant from it - `TX_PME_TIMEOUT_SINCE`, `DEADLOCK_TIMEOUT_SINCE`

Reply:
Agree. I removed the method and the code related to it.